include/cling/Interpreter/Interpreter.h

```diff
@@ -195,6 +196,12 @@ namespace cling {
     ///
     mutable std::vector<ClangInternalState*> m_StoredStates;
 
+    enum {
+      kStdStringTransaction = 0, // Transaction known to contain std::string
+      kNumTransactions
+    };
+    mutable const Transaction* m_CachedTrns[kNumTransactions];
```

Comment:
Could add a comment to explain the choice an array of one element rather than 'just' a pointer?